### PR TITLE
tune Secondary Vertex saving, add LHE root files

### PR DESCRIPTION
**Pull request by arizzi**
in addition the standard cfg now produces both zlib and lzma files

the Vertex producer now allow for selection on non-candidate stuff and produces a candidatePtr with only selected SV for the Simple table producer based on string parser

### CODE REVIEW

File: python/vertices_cff.py [hunk 1]
```diff
@@ -11,15 +11,17 @@
...
     pvName = cms.string("PV"),
     svName = cms.string("SV"),
     svDoc  = cms.string("secondary vertices from IVF algorithm"),
 )
 
 svCandidateTable =  cms.EDProducer("SimpleCandidateFlatTableProducer",
-    src = cms.InputTag("slimmedSecondaryVertices"),
-    cut = cms.string(""), #we should not filter unless we filter also on the vertexTable
+    src = cms.InputTag("vertexTable"),
+    cut = vertexTable.svCut, #this is redundant as the input tag is set to vertexTable
```
Comment (gpetruc) [inline python/vertices_cff.py]: maybe better cut = "" then?

File: python/vertices_cff.py [hunk 2]
```diff
@@ -11,15 +11,17 @@
 vertexTable = cms.EDProducer("VertexTableProducer",
     pvSrc = cms.InputTag("offlineSlimmedPrimaryVertices"),
     svSrc = cms.InputTag("slimmedSecondaryVertices"),
-#    cut = cms.string(""),  #not implemented
+    svCut = cms.string(""),
+    dlenMin = cms.double(0),
+    dlenSigMin = cms.double(5),
```
Comment (gpetruc) [inline python/vertices_cff.py]: can you lower it to 3 sigms

File: test/nano_cfg.py [hunk 3]
```diff
@@ -21,9 +25,16 @@
 
 process.out = cms.OutputModule("NanoAODOutputModule",
     fileName = cms.untracked.string('nano.root'),
-    outputCommands = cms.untracked.vstring("drop *", "keep *_*Table_*_*","keep edmTriggerResults_*_*_*"),
+    outputCommands = cms.untracked.vstring("drop *", "keep FlatTable_*Table_*_*","keep edmTriggerResults_*_*_*"),
```
Comment (gpetruc) [inline test/nano_cfg.py]: this way you drop the x run weights (they're not FlatTable)

Comment (arizzi) [reply test/nano_cfg.py]: should we whitelist the types we can handle downstream?
FlatTables
TriggerResults
MergableCounterTable

that's it?

File: test/nano_cfg.py [hunk 4]
```diff
@@ -21,9 +25,16 @@
...
-    outputCommands = cms.untracked.vstring("drop *", "keep *_*Table_*_*","keep edmTriggerResults_*_*_*"),
+    outputCommands = cms.untracked.vstring("drop *", "keep *Table_*Table_*_*","keep edmTriggerResults_*_*_*"),
    #compressionLevel = cms.untracked.int32(9),
     #compressionAlgorithm = cms.untracked.string("LZMA"),
 
 )
-process.end = cms.EndPath(process.out)  
+process.out1 = cms.OutputModule("NanoAODOutputModule",
+    fileName = cms.untracked.string('lzma.root'),
+    outputCommands = cms.untracked.vstring("drop *", "keep *Table_*Table_*_*","keep edmTriggerResults_*_*_*"),
```
Comment (arizzi) [inline test/nano_cfg.py:35]: is *Table_ enough?

Comment (gpetruc) [reply test/nano_cfg.py:35]: ok

**Pull request merged**